### ISSUE TITLE
Error installing MySQL Client

### ISSUE THREAD

**Issue by joerzepiejewski** [Shopify/heroku-buildpack-mysql#1]
I am trying to use this buildpack and have the following error:
`dpkg-deb: error: '/app/tmp/cache/mysql.deb' is not a debian format archive`

Background:
I have the following buildpacks installed #1 & #2 load fine
1. heroku/ruby
2. https://github.com/gregburek/heroku-buildpack-toolbelt.git
3. https://github.com/Shopify/heroku-buildpack-mysql.git

Sorry for raising an issue here not sure how you'd like Q's

**Comment by grollest** [comment]
@joerzepiejewski I ran into this as well and fixing this under https://github.com/Shopify/heroku-buildpack-mysql/pulls.

Will open follow-up issue to fix the buildpack to always get the current version (or at least fail with a more useful error message).

**Comment by grollest** [comment]
@joerzepiejewski we've temporarily worked around this under #2.

Will be addressed properly under #3.

I'm going to go ahead and close this as you should be unblocked now. Please do reopen if you have any objections.

**Comment by joerzepiejewski** [comment]
@grollest -- thanks for getting this rolling.  👍 I'll give it a shot!  Q: - any tips/insights for using MySQL client and SSH connections?  I'm gonna hit Dr. Google for help - but figured, I'd try you too. 😄

**Comment by grollest** [comment]
@joerzepiejewski this is the code (which I can't take credit for, it's all @SebastianSzturo's goodness) that's using the client if that helps any https://github.com/Shopify/talent-store/pull/1089/files, in particular https://github.com/Shopify/talent-store/blob/3989af17739f9aa62d6c167251f2c6761b9f8d7b/lib/tasks/db.rake#L14-L28

**Comment by joerzepiejewski** [comment]
@grollest thanks - but I don't have access to those repos.  :frowning_face:

**Comment by grollest** [comment]
Ah, I didn't realize this was a public repo! FWIW, here's the code i was pointing at:

```ruby
    dump_production_db_to_file =
      "mysqldump " \
        "-h #{production_db[:host]} " \
        "-u #{production_db[:user]} " \
        "-p#{production_db[:password]} " \
        "#{production_db[:db_name]} " \
        "--single-transaction " \
        "> dump.sql"

    restore_to_staging_db =
      "mysql -h #{staging_db[:host]} " \
        "-u#{staging_db[:user]} " \
        "-p#{staging_db[:password]} " \
        "#{staging_db[:db_name]} " \
        "< dump.sql" \

```

**Comment by joerzepiejewski** [comment]
thx - 🍺

**Comment by mozgbrasil** [comment]
Hello

Good evening

I just added the following URL to buildpacks

Https://github.com/Shopify/heroku-buildpack-mysql.git

When I do the deployment the following error is returned

--


-----> PHP app detected
 !     WARNING: Your 'composer.json' contains a non-'stable' setting
       for 'minimum-stability'. This may cause the installation of
       unstable versions of runtimes and extensions during this deploy.
       It is recommended that you always use stability flags instead,
       even if you have 'prefer-stable' enabled. For more information,
       see https://getcomposer.org/doc/01-basic-usage.md#stability
-----> Bootstrapping...
-----> Installing platform packages...
       - php (5.6.30)
       - ext-gd (bundled with php)
       - apache (2.4.20)
       - nginx (1.8.1)
-----> Installing dependencies...
       Composer version 1.4.1 2017-03-10 09:29:45
       Loading composer repositories with package information
       Installing dependencies from lock file
       Package operations: 2 installs, 0 updates, 0 removals
         - Installing stavarengo/php-sigep-fpdf (dev-master 28f94e5): Loading from cache
         - Installing stavarengo/php-sigep (dev-master d281a0a): Loading from cache
       Generating optimized autoload files
-----> Preparing runtime environment...
       NOTICE: No Procfile, using 'web: heroku-php-apache2'.
-----> Checking for additional extensions to install...
-----> MySQL Client app detected
/app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/lib/build_pack.rb:1:in `require_relative': /app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/lib/build_pack/installer.rb:50: syntax error, unexpected tLABEL, expecting ')' (SyntaxError)
      def run_command_with_message(command:, message:)
                                           ^
/app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/lib/build_pack/installer.rb:74: syntax error, unexpected keyword_end, expecting $end
	from /app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/lib/build_pack.rb:1:in `<top (required)>'
	from /app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/bin/compile:6:in `require_relative'
	from /app/tmp/buildpacks/15b0580d616a9b66dad1c5d4bd63e276920b0d472e5ab68138844907834bb4af83d7d7589c5aeefa63078570f6f346c110731dde32d7bbc18747d0cc38bbf58f/bin/compile:6:in `<main>'
 !     Push rejected, failed to compile MySQL Client app.
 !     Push failed

--

Can you help me

**Comment by drpaulcoleman** [comment]
@mozgbrasil I have a similar issue:

```
> remote: -----> MySQL Client app detected
> remote: -----> Downloading MySQL Client package
> remote:        Looking for clients at: http://security.debian.org/pool/updates/main/m/mysql-5.5/
> remote:        available clients:
> remote:        No suitable clients available
> remote: -----> Failing mysql client installation as no suitable clients were found
> remote:  !     Push rejected, failed to compile MySQL Client app.
> remote: 
> remote:  !     Push failed
```
